src/content/web3/index.md

```diff
@@ -0,0 +1,148 @@
+
+Centralization has helped onboard billions of people to the internet and created the stable, robust infrastructure on which it lives. At the same time, a handful of centralized entities have a stronghold on large swathes of the internet, undemocratically deciding what should and should not be allowed.
+
+Web3 is the answer to this dilemma. Instead of an internet monopolized by large technology companies, web3 embraces decentralization and is being built, operated, and owned by its users. Web3 will put power in the hands of individuals rather than corporations.
+Before we talk about web3, let's explore how we got here.
+
+<Divider />
+
+## The early internet {#early-internet}
+
+Most people think of the internet as a continuous pillar of modern life—it was invented and has just existed since. However, the internet most of us know today is quite different from originally imagined. To understand this better, it's helpful to break the internet's short history into loose periods—web 1.0 and web 2.0.
+
+### Web1: Read-Only {#web1}
+
+In 1989, in CERN, Geneva, Tim Berners-Lee was busy developing the protocols that would become the internet. His idea? To create open, decentralized protocols that allowed information-sharing from anywhere on Earth.
+
+The first inception of the internet, now known as 'Web 1.0', occurred roughly between 1990 to 2004. The internet on Web 1.0 mainly was static websites owned by companies, and there was close to zero interaction between users - individuals seldom produced content - leading to it being known as the read-only web.
+
+### Web2: Read-Write {#web2}
+
+The Web 2.0 period began in 2004 with the emergence of social media platforms. Instead of a read-only, the web evolved to be read-write. Instead of companies providing content to users, they also began to provide platforms to share user-generated content and engage in user-to-user interactions. As more people came online, a handful of top companies began to control a disproportionate amount of the traffic and value generated on the web. Web 2.0 also birthed the advertising-driven revenue model. While users could create content, they didn't own it or benefit from its monetization.
+
+<Divider />
+
+## Web3: Read-Write-Own {#web3}
+
+The premise of 'Web 3.0' was coined by [Ethereum](/what-is-ethereum/) co-founder Gavin Wood shortly after Ethereum launched in 2014. Gavin put into words a solution for a problem that many early crypto adopters felt: the internet required too much trust. That is, most of the internet that people know and use today relies on trusting a handful of private companies to act in the public's best interests.
+
+### What is web3? {#what-is-web3}
+
+Web3 has become a catch-all term for the vision of a new, better internet. At its core, Web3 uses blockchains, cryptocurrencies, and NFTs to give power back to the users in the form of ownership. Chris Dixon said it best: web1 was read-only, web2 is read/write, web3 will be read/write/own.
```

Comment:
I think Chris Dixon was actually retweeting this: 

https://twitter.com/j1mmyeth/status/1459003044067258370?s=20&t=jrElbLC4UYz2chfS35rN-Q

Reply:
I **think** Chris Dixon was the aforementioned thought-leader from the original tweet. Could be wrong. We should verify this. At the very least I would say he popularised it.

Reply:
```suggestion
Web3 has become a catch-all term for the vision of a new, better internet. At its core, Web3 uses blockchains, cryptocurrencies, and NFTs to give power back to the users in the form of ownership. [This tweet](https://twitter.com/j1mmyeth/status/1459003044067258370?s=20&t=jrElbLC4UYz2chfS35rN-Q) said it best: web1 was read-only, web2 is read/write, web3 will be read/write/own.
```